src/binding/python/Attributable.cpp

```diff
@@ -104,12 +104,19 @@ bool setAttributeFromBufferInfo(
         // stride handling
         for( auto d = 0; d < buf.ndim; ++d )
         {
-            // std::cout << "    stride '" << d << "': " << buf.strides[d] << " - " << buf.shape[d] << std::endl;
-            if( buf.strides[d] != buf.shape[d] ) // general criterial
-                if( d > 0 && buf.strides[d] != 1 ) // ok for byte strings
+            // std::cout << "    stride '" << d << "': " << buf.strides[d]
+            //           << " - " << buf.shape[d] << std::endl;
+            if( buf.strides[d] != buf.shape[d] ) // general criteria
```

Comment:
Uh, `strides` in in Bytes: https://docs.scipy.org/doc/numpy/reference/generated/numpy.ndarray.strides.html

Can also simply check the `c_contiguous` flag.